examples/property_prediction/README.md

```diff
@@ -32,7 +32,7 @@ GCN-VN          | 4.9M    | 0.1225    | 0.1485    |
 GIN-VN          | 6.7M    | 0.1150    | 0.1395    |
 Graphormer-Small| 12.5M   | 0.0778    | 0.1264    |
 Graphormer   | 47.1M   | 0.0582    | 0.1234    |
-Graphormer-v2   | 47.1M   | 0.0309    | **0.1201**    |
+Graphormer-v2   | 48.3M   | 0.0309    | **0.1201**    |
```

Comment:
May report consistent number with our paper?

Reply:
In paper, the number of parameters is 48.3M for base models. I think 47.1M is for Graphormer V1.

Reply:
I mean the MAE result

Reply:
Ok. I'll fix that.